pyhelm/chartbuilder.py

```diff
@@ -236,6 +233,15 @@ def get_helm_chart(self):
         self._helm_chart = helm_chart
         return helm_chart
 
+    def read_file(self, path):
```

Comment:
This can (and should) be a static method

Reply:
No worries. Using a decorator like this you mean?
```python
    @staticmethod
    def read_file(self, path):
```

Reply:
I've gone with this and added to the last commit. Let me know if that works for you